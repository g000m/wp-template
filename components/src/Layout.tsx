import React, { FC, ReactChild } from 'react';

export type LayoutProps = {children:ReactChild};

const Layout: FC<LayoutProps> = ({children}) => {
    return (
        <div>{children}</div>
    )
};

export  default  Layout;