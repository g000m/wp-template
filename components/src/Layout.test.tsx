import * as React from "react";
import { render,
   // fireEvent,
   // act
} from '@testing-library/react';
import Layout  from "./Layout";

describe( 'Layout', () => {
    it('Lays it out', () => {
        const { container, getByText } = render(
            <Layout>Hi Roy</Layout>
        );
        expect(getByText('Hi Roy')).toBeTruthy();
        expect(container).toMatchSnapshot();
    });
})