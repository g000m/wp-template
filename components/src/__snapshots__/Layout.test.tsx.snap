// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Lays it out 1`] = `
<div>
  <div>
    Hi Roy
  </div>
</div>
`;
